packages/sdk/src/base/auth.ts

```diff
@@ -131,22 +137,21 @@ export class Auth implements IAuth {
 		return await this.refresher.debounce(force);
 	}
 
-	async login(credentials: AuthCredentials, options?: Partial<AuthLoginOptions>): Promise<AuthResult> {
-		options = options || {};
-		const response = await this.transport.post<AuthResult>(
-			'/auth/login',
-			{
-				mode: this.options.mode,
-				...credentials,
-			},
-			{
-				refreshTokenIfNeeded: false,
-				sendAuthorizationHeaders: false,
-			}
-		);
+	async login(credentials: AuthCredentials, refreshOptions?: Partial<AuthAutoRefreshOptions>): Promise<AuthResult> {
+		// why does login take its own refresh options object so you can change the global login options on repeat logins?
+		refreshOptions = refreshOptions || {};
+
+		// Shouldn't be sending any credentials on login
```

Comment:
I added this to remove an option that was only used here in the axios transport, but I think there's a situation where this login request is happening while another request is also firing, and the timing works out so that the token is deleted and the other request can no longer use it. I could be misunderstanding how the single-thread nature of js works though.